llmnrd.c

```diff
@@ -332,7 +349,11 @@ int main(int argc, char **argv)
 		close(llmnrd_sock_ipv6);
 	if (llmnrd_sock_ipv4 >= 0)
 		close(llmnrd_sock_ipv4);
-	free(hostname);
+	for(name_i = 0; name_i < name_count; ++name_i) {
```

Comment:
Add a space after the `for` keyword:

```suggestion
	for (name_i = 0; name_i < name_count; ++name_i) {
```